tests/test_app.py

```diff
@@ -0,0 +1,21 @@
+"""Tests for the Streamlit app using AppTest framework."""
+
+from streamlit.testing.v1 import AppTest
+
+
+def run_wrapper() -> None:
+    from lecture_feedback.app import run  # noqa: PLC0415
```

Comment:
```suggestion
    app = AppTest.from_file("src/lecture_feedback/app.py")
```

Also in other locations

Reply:
at least we are now sure that this is probably the best way